src/index.ts

```diff
@@ -30,6 +30,8 @@ export function obfuscator(options: RollupObfuscatorOptions = {}): Plugin {
 
 	return {
 		name: 'obfuscator',
+		apply: 'build',
+		enforce: 'post',
```

Comment:
The typecheck fails because the `Plugin` type by `rollup` does not know about `apply` and `enforce`.

Reply:
Probably best to type it custom, not worth adding another dep for typing it better

```ts
interface ObfuscatorPlugin extends Plugin {
	enforce: string;
}
```

Reply:
Good idea! I have added such a type to my draft.